_templates/menu01.html

```diff
@@ -14,6 +14,7 @@
       <li><a href="{{pathto('products/SrMise')}}">SrMise</a></li>
       <li><a href="{{pathto('products/mPDF')}}">mPDF</a></li>
       <li><a href="{{pathto('products/xinterpdf')}}">xINTERPDF</a></li>
+      <li><a href="{{pathto('products/utils')}}">diffpy.utils</a></li>
```

Comment:
This is a python package so should maybe be under "python packages" sub-list, but maybe we should discuss this a bit?